pkg/helpers/types.go

```diff
@@ -28,5 +28,7 @@ type TestOptions struct {
 	Metrics         map[string]*vegeta.Metrics     // Stores results from each Test
 	Attacker        *vegeta.Attacker
 	OutputDirectory string // Directory to write vegeta results as a File
-	Connection      *sdk.Connection
+	// TODO make cofigurable???
```

Comment:
It would not be a bad idea but not super important for now. We can always add that in later.